src/lib/status-utils.js

```
@@ -0,0 +1,306 @@
+  }, {});
+}
+
+/**
+ * Calculate AI interaction metrics
+ * @param {object[]} tasks List of tasks
+ * @return {object|null} AI metrics object or null if no AI interactions
+ */
+function calculateAIMetrics(tasks) {
+  const aiInteractions = tasks.filter(task =>
+    task.metadata.tags &&
+    task.metadata.tags.includes('ai-interaction')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)